pkg/apis/clusterregistry/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -82,41 +82,41 @@ var (
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
 					Type: "object",
 					Properties: map[string]v1beta1.JSONSchemaProps{
-						"apiVersion": v1beta1.JSONSchemaProps{
+						"apiVersion": {
```

Comment:
It seems like `kubebuilder` needs to be optimized to produce go-vetted code in this case.

Reply:
Yes! Filed https://github.com/kubernetes-sigs/kubebuilder/issues/171